data/json/items/containers.json

```diff
@@ -3500,5 +3500,21 @@
       }
     ],
     "flags": [ "COLLAPSE_CONTENTS" ]
+  },
+  {
+    "id": "duct_tape_roll",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "duct tape roll" },
+    "description": "A small cardboard roll with duct tape rolled around it.",
```

Comment:
I would make this a generic "tape roll" "cardboard roll", not sure how you refer to those in english, that way it'd be easy to swap the contents for medical tape, toilet paper etc

also the `other` category would fit better

Reply:
To be fair - you're right! On both accounts. I was very close to naming it just a cardboard roll but I was worried that this way people might get confused. Though if we are going to reuse it, as we should admittedly, then this is the right approach. I'll correct those tomorrow once the tests inevitably yell at me for forgetting to remove duct tape from the known bad density list.